.github/workflows/pytest.yml

```
@@ -21,3 +21,24 @@ jobs:
         sudo mkdir /var/log/s3_upload
         sudo chmod o+w /var/log/s3_upload
         pytest -v --cov --count 10 --random-order tests/unit/
+    - name: Get test coverage %
+      run: |
```

actionlint (1.7.3)

shellcheck reported issue in this script: SC2155:warning:2:8: Declare and assign separately to avoid masking return values

(shellcheck)